docs/_static/vdr/v1.yaml

```diff
@@ -100,7 +100,32 @@ paths:
             text/plain:
               schema:
                 type: string
-
+    delete:
+      summary: Deactivatates a Nuts DID Document according to the specification.
+      operationId: "deactivateDID"
+      tags:
+        - DID
+      responses:
+        "200":
+          description: DID Document has been deactivated.
+        "400":
+          description: DID couldn't be deactivated because the input conflicts or is malformed.
```

Comment:
conflict = HTTP 409

Reply:
409 is rare and I would not recommend using it.

Reply:
I disagree but let's agree to disagree...? There's not tens of common HTTP codes for us to be just using 200, 301, 400 and 500...